sphinx/setup_command.py

```diff
@@ -105,7 +105,8 @@ def initialize_options(self):
         self.config_dir = None  # type: unicode
         self.link_index = False
         self.copyright = ''
-        self.verbosity = 0
+        # Link verbosity to distutils' (which uses 1 by default).
+        self.verbosity = self.distribution.verbose - 1
```

Comment:
Are `self.distribution` and `self.distribution.verbose` public API? It seems there are no document for them.

Reply:
It seems distlib provides `Command.distribution` and `Distribution.verbose` since py27 (I just checked in GitHub).
https://github.com/python/cpython/blob/master/Lib/distutils/cmd.py#L61
https://github.com/python/cpython/blob/3.5/Lib/distutils/dist.py#L134

Unfortunately they are not documented. But I suppose it's okay.
https://docs.python.org/3/distutils/apiref.html